guides/Automation/Steel.md

```diff
@@ -0,0 +1,19 @@
+# Steel
+
+Steel is one of the most important resources throughout earlygame. You will need a lot of it, so it's useful to try to make a system to get it quickly.
+
+### Blast Furnace
+
+The first way of obtaining steel is thorugh the Blast Furnace, from Chapter 4. There is no way to automate the basic blast furnace, however, its successor, the Reinforced Blast Furnace, is automatable.
+
+![Improved Blast Furnace](/guides/files/Steel/blastfurnace.png)
+
+As you see in the image, the Coal Coke and Iron go in the top, Steel comes out the front, and the leftover Slag goes out the back.
+
+### Metallurgic Infuser
+
+The Metallurgic Infuser is first obtained in Chapter 6. It takes one item and "infuses" it with another item to create something new. Aside form it's other functions, the Metallurgic Infuser can be used to make a lot of steel very quickly.
+
+![Steel Setup](/guides/files/Steel/infusionsetup.png)
+
+Note: Using the Enrichment Chamber is completely optional, but i reccomend it, as it gets 8x the infusion power out of your Coal than just putting the Coal directly in the Infusers.

```

Comment:
try to write in 3rd person - instead of "i", "it is recommended" 
reccomend -> recommend
infusion power -> infusion amount